### PR TITLE
Update table resizer svg and bump tokens package

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

## 🧢 Your Project:

RSP

### CODE REVIEW

**Comment by LFDanLu** [comment]
Will run chromatic soon, need to double check that the tokens bump is what we expect

**Review by reidbarber — DISMISSED** [review]
LGTM

File: packages/@react-spectrum/s2/src/Badge.tsx [hunk 1]
```diff
@@ -165,12 +165,12 @@ const badge = style<BadgeStyleProps>({
     fillStyle: {
       outline: {
         variant: {
-          accent: 'accent-800',
-          informative: 'informative-800',
-          neutral: 'gray-700',
-          positive: 'positive-700',
-          notice: 'notice-700',
-          negative: 'negative-800'
+          accent: lightDark('blue-800', 'blue-900'), // accent-visual-color
```
Comment (yihuiliao) [inline packages/@react-spectrum/s2/src/Badge.tsx:168]: is there are reason why we aren't using 'accent-800'/'accent-900' instead? that would more closely match how they describe it in the token figma file

Comment (LFDanLu) [reply packages/@react-spectrum/s2/src/Badge.tsx:168]: good call! I'm not sure why I dove down an extra layer when getting the colors haha

File: packages/@react-spectrum/s2/src/Badge.tsx [hunk 2]
```diff
@@ -165,12 +165,12 @@ const badge = style<BadgeStyleProps>({
...
       outline: {
         variant: {
-          accent: 'accent-800',
-          informative: 'informative-800',
-          neutral: 'gray-700',
-          positive: 'positive-700',
-          notice: 'notice-700',
-          negative: 'negative-800'
+          accent: lightDark('accent-800', 'accent-900'), // accent-visual-color
+          informative: lightDark('accent-800', 'accent-900'), // informative-visual-color
```
Comment (yihuiliao) [inline packages/@react-spectrum/s2/src/Badge.tsx:169]: mega nit: i think this should be `informative-800` and `informative-900` instead of `accent`

**Review by snowystinger — DISMISSED** [review]

**Review by reidbarber — DISMISSED** [review]

**Review by yihuiliao — APPROVED** [review]

**Review by snowystinger — APPROVED** [review]

**Pull request merged**